flux-templates/.azuredevops/onboard-new-service.yaml

```diff
@@ -0,0 +1,43 @@
+name: $(BuildID)
+
+parameters:
+  - name: onboardingJson
+    displayName: "Json structure to OnBoard new project"
+    type: string
+
+trigger: none
+
+pool:
+  vmImage: windows-latest
+
+resources:
+  repositories:
+    - repository: AdpFluxServices
+      name: defra/adp-flux-services
+      endpoint: DEFRA
+      type: github
+      ref: main
+
+steps:
+  - checkout: self
+    path: s/source
+    persistCredentials: true
+
+  - checkout: AdpFluxServices
+    path: s/AdpFluxServices
+    persistCredentials: true
+
+  - task: PowerShell@2
+    displayName: 'Add new project manifests'
+    # condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/main'), eq('${{ parameters.publishFromFeature }}', true)))
```

Comment:
@aadev1  is this condition required?

Reply:
That condition isn't required.  I copied it from another pipeline and forgot to remove the comment.  I've taken it out now.